ui/component/txoList/view.jsx

```diff
@@ -0,0 +1,264 @@
+// @flow
+import React, { useEffect } from 'react';
+import { withRouter } from 'react-router';
+
+import { TX_LIST } from 'lbry-redux';
+import * as PAGES from 'constants/pages';
+import * as TXO from 'constants/txo_list';
+
+import TransactionListTable from 'component/transactionListTable';
+import Paginate from 'component/common/paginate';
+// import FileExporter from '../common/file-exporter'; // reimplement as modal
```

Comment:
if it's a lot of work, don't worry about adding this back for the first implementation. We can add it back once the dust settles